src/client/Microsoft.Identity.Client/TelemetryCore/Internal/HttpTelemetryContent.cs

```diff
@@ -18,49 +20,81 @@ public HttpTelemetryContent(EventBase evt)
                 evt.TryGetValue(MsalTelemetryBlobEventNames.ApiErrorCodeConstStrKey, out string errorCode);
                 evt.TryGetValue(MsalTelemetryBlobEventNames.ForceRefreshId, out string forceRefresh);
 
-                ApiId = apiId ?? string.Empty;
-                CorrelationId = correlationId ?? string.Empty;
-                LastErrorCode = errorCode ?? string.Empty;
+                ApiId.Add(apiId ?? string.Empty);
```

Comment:
Should these properties remain as strings rather than lists? As far as I saw, HttpTelemetryContent is instantiated for each EventBase instance and nothing is ever added to these lists besides in here.